src/js/xhr.js

```diff
@@ -1,5 +1,6 @@
-import * as VjsUtils from './util';
```

Comment:
Probably a wrapper using the external module. Though, is `file://` support useful? `withCredential` support is available.